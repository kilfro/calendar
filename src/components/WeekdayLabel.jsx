import React from 'react'
import '@style/day.less'
import '@style/week.less'

const WeekdayLabel = () => {
    const labels = ['ПН', 'ВТ', 'СР', 'ЧТ', 'ПТ', 'СБ', 'ВС']
    return (
        <div className='week label'>
            {labels.map(lab =>
                <div className='day label' key={lab}>{lab}</div>
            )}
        </div>
    )
}

export default WeekdayLabel