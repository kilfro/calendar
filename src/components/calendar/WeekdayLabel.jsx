import React from 'react'
import '@style/calendar/day.less'
import '@style/calendar/week.less'

const WeekdayLabel = () => {
    const labels = ['ПН', 'ВТ', 'СР', 'ЧТ', 'ПТ', 'СБ', 'ВС']
    return (
        <div className='week label'>
            {labels.map(lab =>
                <div className='day label' key={lab}>{lab}</div>
            )}
        </div>
    )
}

export default WeekdayLabel