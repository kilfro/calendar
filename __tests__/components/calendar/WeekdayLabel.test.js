import React from 'react'
import { shallow } from 'enzyme'
import WeekdayLabel from '../../../src/components/calendar/WeekdayLabel'

describe('WeekdayLabel', () => {
    test('should render correctly', () => {
        const component = shallow(<WeekdayLabel />)

        expect(component).toMatchSnapshot()
    })
})
