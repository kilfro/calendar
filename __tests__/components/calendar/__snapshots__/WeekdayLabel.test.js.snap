// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekdayLabel should render correctly 1`] = `
<div
  className="week label"
>
  <div
    className="day label"
    key="ПН"
  >
    ПН
  </div>
  <div
    className="day label"
    key="ВТ"
  >
    ВТ
  </div>
  <div
    className="day label"
    key="СР"
  >
    СР
  </div>
  <div
    className="day label"
    key="ЧТ"
  >
    ЧТ
  </div>
  <div
    className="day label"
    key="ПТ"
  >
    ПТ
  </div>
  <div
    className="day label"
    key="СБ"
  >
    СБ
  </div>
  <div
    className="day label"
    key="ВС"
  >
    ВС
  </div>
</div>
`;
